src/Listeners/UptimeCheckRecoveredListener.php

```diff
@@ -4,16 +4,27 @@
 
 use Illuminate\Support\Facades\Notification;
 use Taecontrol\MoonGuard\Repositories\UserRepository;
+use Taecontrol\MoonGuard\Notifications\SlackNotifiable;
 use Taecontrol\MoonGuard\Events\UptimeCheckRecoveredEvent;
 use Taecontrol\MoonGuard\Notifications\UptimeCheckRecoveredNotification;
 
 class UptimeCheckRecoveredListener
 {
     public function handle(UptimeCheckRecoveredEvent $event): void
     {
-        Notification::send(
-            UserRepository::all(),
-            new UptimeCheckRecoveredNotification($event->uptimeCheck, $event->downtimePeriod)
-        );
+        $channels = config('moonguard.notifications.channels');
+
+        foreach ($channels as $channel) {
+            $users = ($channel === 'slack') ? new SlackNotifiable() : UserRepository::all();
```

Comment:
Rename this variable into:

```suggestion
            $notifiables = ($channel === 'slack') ? new SlackNotifiable() : UserRepository::all();
```